backend/tests/Designer.Tests/Controllers/ApiTests/DesignerEndpointsTestsBase.cs

```diff
@@ -118,6 +118,25 @@ protected async Task CopyOrgRepositoryForTest(string developer, string org, stri
             TestOrgPath = await TestDataHelper.CopyOrgForTest(developer, org, repo, targetOrg, targetRepository);
         }
 
+        /// <summary>
+        /// Copies a repository from the test repositories to a temporary location for testing.
+        /// Limitation is that test org has to be created through <see cref="CopyOrgRepositoryForTest"/>
+        /// </summary>
```

Comment:
Foreslår en litt annen formulering:
```suggestion
        /// <summary>
        /// Copies a repository from test data to a temporary location for testing.
        /// </summary>
        /// <remarks>
        /// <see cref="CopyOrgRepositoryForTest"/> must be used first.
        /// </remarks>
```

Reply:
I've adjusted the text a bit to be the same as the start as the function `CopyRepositoryForTest` on line 73

```cs
        /// <summary>
        /// Copies a repository from the test repositories to a temporary location for testing.
        /// </summary>
        /// <param name="developer">Username of developer.</param>
        /// <param name="org">Organisation short name.</param>
        /// <param name="repo">Repository name.</param>
        /// <param name="targetOrg">Test organisation name.</param>
        /// <param name="targetRepository">test repository name.</param>
        /// <exception cref="InvalidOperationException"></exception>
        /// <remarks>
        /// <see cref="CopyOrgRepositoryForTest"/> must be used first.
        /// </remarks>
```